Misc/NEWS.d/next/Library/2024-09-11-16-40-12.gh-issue-123963.TzzIY8.rst

```diff
@@ -0,0 +1 @@
+Expose the CurrentByteCount field from expat parsers
```

Comment:
```suggestion
Expose the :attr:`xmlparser.CurrentByteCount` field for
:mod:`Expat XML <xml.parsers.expat>` parsers.
Patch by Jesper Jensen.
```

Reply:
If you would like to, you can use the alternative form:
```
Contributed by Jesper Jensen in :gh:`123963`
```
(feel free to ignore this comment)

Reply:
I only saw this kind of wording in What's New entries so I'm not sure if we do it for the simple news entries.

Reply:
I did as the comment suggested and ignore it :P